doc/content/microreactors/s8er/s8er_multiphysics_model.md

```diff
@@ -0,0 +1,245 @@
+
+### Geometry
+
+The main tools used to generate this mesh were MOOSE based meshgenerators, primarily the [PolygonConcentricCircleMeshGenerator](https://mooseframework.inl.gov/source/meshgenerators/PolygonConcentricCircleMeshGenerator.html) for a fuel channel, [HexIDPatternMeshGenerator](https://mooseframework.inl.gov/source/meshgenerators/HexIDPatternedMeshGenerator.html) for a lattice of fuel channels, the [PeripheralRingMeshGenerator](https://mooseframework.inl.gov/source/meshgenerators/PeripheralRingMeshGenerator.html) for the internal reflector and barrel, and the [PlaneDeletionGenerator](https://mooseframework.inl.gov/source/meshgenerators/PlaneDeletionGenerator.html) for adjustments to the geometry.
+
+!listing microreactors/s8er/core_2D_griffin_coupled.i block=Mesh
+
+1. 9 individual fuel channels are modeled, 1 for each corresponding radial ring 
+2. The fuel channels are mapped into the lattice in the appropiate arrangement from the homogenized universe data.
+3. A peripheral ring is enclosed around the lattice to represent the internal reflector + core barrel (last radial layer)
+4. The mesh is cut along the ring at various points to preserve the geometry and dimension of the reference serpent model. 
+5. The blocks are mapped to their appropiate material ID.
+
+A table with corresponding mapping between the Serpent Universe ID and the Griffin Block ID is shown below.
+
+!table id=Homogenized Universe Mapping caption= Homogenized Universe Mapping
+| Serpent Universe ID | Material ID | Block ID | Material/Region |
+| -------- | ------- | ----- | ------------ |
+| 100      |   100   |  1    | Fuel Channel |
+| 200      |   200   |  2    | Fuel Channel |
+| 900      |   900   |  3    | Fuel Channel |
+| 400      |   400   |  4    | Fuel Channel |
+| 500      |   500   |  5    | Fuel Channel |
+| 600      |   600   |  6    | Fuel Channel |
+| 700      |   700   |  7    | Fuel Channel |
+| 800      |   800   |  8    | Fuel Channel |
+| 300      |   300   |  9, 10 | Coolant Channel |
+| 1200     |   1200  |  11   | Internal Reflector + Core Barrel |
+
+!media s8er/griffin_universe_map.png
+  caption= S8ER Griffin Mesh, from [!citep](s8er_naupa2022)
+  style=width:100%;margin-left:auto;margin-right:auto
+
+### TransportSystems
+
+As stated previously, the Griffin model uses the newly optimized DFEM-SN transport scheme to solve for the eigenvalue. Here a 16 group energy structure is used with vacuum boundary conditions on the outeredge of the core barrel. The solver uses 9 azimuthal and 3 polar angles, typically the default for these are 6 and 2 respectively but because this highly anistropic, a higher order of angles is required for solution fidelity. The benefit with using the DFEM-SN scheme rather than the stand CFEM-Diffusion scheme, is the removal of several layers of calculation for the same fidelity, however at the expense of a more expensive computation. The DFEM-SN scheme is optimized and scales well with problem size. 
```

Comment:
```suggestion
As stated previously, the Griffin model uses the newly optimized DFEM-SN transport scheme to solve for the eigenvalue. Here a 16 group energy structure is used with vacuum boundary conditions on the outer edge of the core barrel. The solver uses 9 azimuthal and 3 polar angles, typically the default for these are 6 and 2 respectively but because this system is highly anisotropic, a higher order of angles is required for solution fidelity. The benefit with using the DFEM-SN scheme rather than the stand CFEM-Diffusion scheme, is the removal of several layers of calculation for the same fidelity, however at the expense of a more expensive computation. The DFEM-SN scheme is optimized and scales well with problem size. 
```

Reply:
maybe per quarter? it s likely 6 polar in 3D not 3, and 36 azimuthal?